flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/FromElementsFunction.java

```diff
@@ -65,8 +72,11 @@
 @Deprecated
 @PublicEvolving
 public class FromElementsFunction<T>
-        implements SourceFunction<T>, CheckpointedFunction, OutputTypeConfigurable<T> {
-
+        implements SourceFunction<T>,
+                CheckpointedFunction,
+                OutputTypeConfigurable<T>,
+                LineageVertexProvider {
+    private static final String LINEAGE_NAMESPACE = "values://FromElementsFunction";
```

Comment:
`FromElementsFunction` is a deprecated class, could you move `LINEAGE_NAMESPACE` to another class instead of `FromElementsFunction`?

Reply:
It is not uniformed. I would like to introduce the convention from open lineage. Please see the example https://openlineage.io/docs/spec/naming/. 

After this PR, I will going to create a separate PR for the doc to introduce suggested naming convenient in Flink Docs. How do you think?